src/mam4xx/ndrop.hpp

```diff
@@ -1,105 +1,218 @@
 #ifndef MAM4XX_NDROP_HPP
 #define MAM4XX_NDROP_HPP
 
-#include <haero/aero_species.hpp>
+#include <ekat/util/ekat_math_utils.hpp>
+
 #include <haero/atmosphere.hpp>
-#include <haero/constants.hpp>
 #include <haero/math.hpp>
 
 #include <mam4xx/aero_config.hpp>
 #include <mam4xx/conversions.hpp>
 #include <mam4xx/mam4_types.hpp>
+#include <mam4xx/ndrop.hpp>
```

Comment:
Is this file including itself?

Slightly related: can you add a line to `src/CMakeLists` to make sure that the `ndrop.hpp` header is installed when we run `make install`?

Reply:
Done.